src/sast_agent_workflow/tools/pre_process.py

```diff
@@ -34,33 +38,50 @@ async def pre_process(
     
     logger.info("Initializing Pre_Process function...")
     
-    def _temp_create_basic_sast_tracker_for_testing() -> SASTWorkflowTracker:
-        """Just a temporary function for testing to create a SASTWorkflowTracker from a JSON file path"""
-        logger.debug("Converting TextIOWrapper to SASTWorkflowTracker")
-        try:        
-            # Check if config field exists, if not create it
-            if 'config' not in temp_data_for_testing:
-                logger.info("Missing 'config' field in JSON, creating config object")
-                temp_data_for_testing['config'] = Config()
-            
-            # Now create SASTWorkflowTracker with the complete data
-            return SASTWorkflowTracker.model_validate(temp_data_for_testing)
-        except Exception as e:
-            logger.error(f"Failed to create SASTWorkflowTracker from temp_data_for_testing: {e}")
-            raise
-
-
+    def _create_default_analysis_response() -> AnalysisResponse:
+        default_field_value = "This is a default value, if it's not replaced, something went wrong"
```

Comment:
#Clarification 
this is meant for manual result checking for failures right?
are we gonna use it in the write_Results ,where if we had an unchanged value we mark the analysis as failed?